ops/testing.py

```diff
@@ -1890,6 +1890,42 @@ def run_action(self, action_name: str,
                 output=action_under_test.output)
         return action_under_test.output
 
+    def set_cloud_spec(self, content: Dict[str, str]):
+        """Set cloud specification (metadata) including credentials.
+
+        Call this method before trying to call `Harness.Model.get_cloud_spec`.
```

Comment:
This should be a ReST link, like this but I'm not sure if I have the reference totally right so please check that :smile: :

```suggestion
        Call this method before trying to call :meth:`Model.get_cloud_spec`.
```